herbie_core/views/save_business_entity_view.py

```diff
@@ -39,7 +41,10 @@ def post(self, request: Request, business_entity: str) -> Response:
         version = body[Constants.VERSION]
         key = body[Constants.KEY]
         payload = body[Constants.PAYLOAD]
-        error_messages = self._validator.validate_schema(payload, business_entity, version)
+
+        schema = self._get_json_schema(business_entity=business_entity, version=version)
+
+        error_messages = self._validator.validate_schema(schema, payload, business_entity, version)
```

Comment:
:bulb: We could have two methods for validating the schema, one that is expecting the schema as an argument and one that does not. What do you think?